content/en/docs/reference/setup-tools/kubeadm/implementation-details.md

```diff
@@ -372,7 +368,7 @@ As soon as the control plane is available, kubeadm executes following actions:
 
 Please note that:
 
-1. Mark master phase can be invoked individually with the [`kubeadm alpha phase mark-master`](/docs/reference/setup-tools/kubeadm/kubeadm-alpha/#cmd-phase-mark-master) command
+1. Mark master phase can be invoked individually with the [`kubeadm init phase mark-control-plane`](/docs/reference/setup-tools/kubeadm/kubeadm-alpha/#cmd-phase-mark-master) command
```

Comment:
`kubeadm init phase mark-control-plane all`

Reply:
it should be without `all`. it's just one command.